update.py

```diff
@@ -193,17 +194,20 @@ def fetchlogmessages(site=None, cache=None):
 
 
 def build_one(wiki, fast):
-    '''build one wiki'''
-    debug('Using make for sphinx: %s' % wiki)
-    if platform.system() == "Windows":
-        # This will fail if there's no folder to clean, so no check_call here
-        if not fast:
-            subprocess.run(["make.bat", "clean"], cwd=wiki, shell=True)
-        subprocess.check_call(["make.bat", "html"], cwd=wiki, shell=True)
-    else:
-        if not fast:
-            subprocess.check_call(["nice", "make", "clean"], cwd=wiki)
-        subprocess.check_call(["nice", "make", "html"], cwd=wiki)
+    """build one wiki"""
+    print('Using sphinx-build for sphinx: %s' % wiki)
+
+    source_dir = os.path.join(wiki, 'source')
+    output_dir = os.path.join(wiki, 'build')
+    html_dir = os.path.join(output_dir, 'html')
+    doctree_dir = os.path.join(output_dir, 'doctrees')
+
+    # This will fail if there's no folder to clean, so we check first
+    if not fast and os.path.exists(output_dir):
+        shutil.rmtree(output_dir)
+
+    app = Sphinx(source_dir, source_dir, html_dir, doctree_dir, 'html', parallel=2)
```

Comment:
What made you choose a value of `2` for `parallel`?

If you use named arguments for all arguments, it could improve readability. Right now, it's not clear why you pass in `source_dir` twice. I found docs here: 
https://smobsc.readthedocs.io/en/stable/_modules/sphinx/application.html#Sphinx

Reply:
yep right, will update